osrf/vrx
# New ball shooter plugin for the "Scan and Dock and Deliver" task

See issue #280.

How to test it:

Terminal 1:
```
roslaunch vrx_gazebo vrx.launch
```

Terminal 2:
```
rostopic pub /wamv/shooters/ball_shooter/fire std_msgs/Empty "{}" --once
```

You should observe how the blue ball is fired.
![ball_shooter](https://user-images.githubusercontent.com/1440739/128942716-c593cabe-8e99-4d42-aa3e-4f507dedd510.gif)


Feel free to play with the plugin parameters in `ball_shooter.xacro`  for testing.

Note about the ball shooter model: For now, the model is just a box. This will be replaced in the future when a nicer model is ready. It shouldn't be any functional difference in any case.
